nodl_python/nodl/errors.py

```diff
@@ -22,6 +22,10 @@ class NoDLError(Exception):
     """Base class for all NoDL exceptions."""
 
 
+# Parsing Errors
+# region
```

Comment:
Heh?

Reply:
See the vscode comment below
fixed in 97698c7